pkg/ocm/clusters.go

```diff
@@ -174,6 +174,15 @@ type Spec struct {
 
 	// Control Plane Machine Pool attributes
 	AdditionalControlPlaneSecurityGroupIds []string
+
+	// Registry Config
+	AllowedRegistries          []string
+	BlockedRegistries          []string
+	InsecureRegistries         []string
+	AllowedRegistriesForImport string
+	PlatformAllowlist          string
+	AdditionalTrustedCa        string
+	ClusterRegistryConfig      *cmv1.ClusterRegistryConfigBuilder
```

Comment:
Does it make sense to include the builder in this spec?
I feel like either the builder or the attributes should be enough :thinking:

Reply:
Removed the builder